lfapi/models.py

```diff
@@ -200,19 +198,38 @@ def to_csv(self, fp=None, delimiter=','):
 
     return fp.getvalue() if isinstance(fp, io.StringIO) else None
 
+  @depends_on('pandas')
   def to_pandas(self):
     """Convert the model to a Pandas DataFrame. Not implemented if Pandas is
     not installed.
     """
-    if pd is None:
-      raise NotImplementedError('Pandas is not installed.')
-
     rows = self.as_dict_list()
     return pd.DataFrame(
       columns=self._labels,
       data=rows
     )
 
+  @depends_on('pyarrow')
+  def to_pyarrow(self):
+    """Convert the model to a PyArrow Table. Not implemented if PyArrow is not
+    installed.
+    """
+    rows = self.as_dict_list()
+    return pa.Table.from_pylist(rows)
+
+  @depends_on('pyarrow.parquet')
+  def to_parquet(self, where, **pq_kwargs):
+    """Send the model to a Parquet file.
+
+    Arguments:
+    where
+      the filename or pyarrow.NativeFile instance
+    **pq_kwargs
+      accepts any keyword arguments supported by pq.write_table()
+    """
+    table = self.to_pyarrow()
+    pq.write_table(table, where, **pq_kwargs)
```

Comment:
seems odd that other `to_X` methods return something whereas this method `to_parquet` is just a side effect

`write_X` does seem to express intent that it would be a side-effect by convention (it writes), whereas `to_X` sounds like it would convert an object from one medium to another (in-memory).